packages/common/src/utils/types.ts

```diff
@@ -11,24 +14,22 @@ export enum SUBMISSION_STATE {
 }
 
 /**
- * Used as a Response type on a Create new Active Submission
+ * Enum used in the Reponse on Create new Submissions
  */
-export type CreateSubmissionResult = {
-	readonly successful: boolean;
-	readonly submission: CreateActiveSubmission;
-	batchErrors: DeepReadonly<BatchError>[];
-};
+export enum CREATE_SUBMISSION_STATE {
+	PROCESSING = 'PROCESSING',
+	INVALID_SUBMISSION = 'INVALID_SUBMISSION',
+	PARTIAL_SUBMISSION = 'PARTIAL_SUBMISSION',
+}
```

Comment:
I'll provide a code example after I get through the code review... I would like to use TS `enum` as little as possible. As you've written this there are no problems, but it is the only TS code that gets compiled into JS code accessible at run time. This has a few potential weird points. Can leave this for now but remind me to give a JS alternative that provides the same types and values.

Reply:
replaced all typescript enums to use const objects and types instead